GTG/plugins/send_email/sendEmail.py

```diff
@@ -32,22 +33,38 @@ class pluginSendEmail:
     """
     The plugin.
     """
+    def __init__(self):
+        """
+        Initialize variables.
+        """
+        self.browser_plugin_api = None
+        self.taskview_plugin_api = None
```

Comment:
No the problem is example we are in editor and user deactivates the plugin by going to preferences. Then we have to remove buttons from both editor as well as the the main manager. The problem is the engine sends API of editor, not of manager.